utils/cli/core/core.py

```diff
@@ -34,19 +34,13 @@ def tasks_data(self, task_id, resource_type, resources, **kwargs):
             data = {'remote_files[{}]'.format(i): f for i, f in enumerate(resources)}
         elif resource_type == ResourceType.SHARE:
             data = {'server_files[{}]'.format(i): f for i, f in enumerate(resources)}
-        data['image_quality'] = 50
```

Comment:
CLI tests failed because there was no default value specified for this field, so please don't remove it: https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/engine/serializers.py#L305